### PR TITLE
Login signup

### PR DESCRIPTION
**Pull request by dbu329**

### CODE REVIEW

File: app/components/LoginForm/index.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  Label,
+  Button,
+  InputGroup,
+  Input,
+  Icon,
+  View,
+  Spinner,
+  Text
+} from 'native-base'
+import FontAwesomeIcon from 'react-native-vector-icons/FontAwesome';
```
Comment (n-trance) [inline app/components/LoginForm/index.js]: imports and document overall, we're either going to have ; or no ; currently there's none. Let's stick with that OR change from now.

File: app/components/LoginForm/index.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+import { Alert } from 'react-native'
+import { inject, observer } from 'mobx-react/native'
+import Styles from './style.js';
+
+
+@inject('mainStore') @observer
```
Comment (n-trance) [inline app/components/LoginForm/index.js]: more of just styling, just for readability if we add more decorators

@inject('mainstore')
@observer
export.... etc

File: app/components/LoginForm/index.js [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+              onChangeText={(pass) => { this.updatePassword(pass)}} />
+          </InputGroup>
+          <Button rounded block success
+            style={Styles.loginButton}
+            onPress={this.signIn.bind(this)}>
+            <Text>Login</Text>
+          </Button>
+          <View style={Styles.forgotPasswordContainer}>
+            <Button transparent
+              onPress={this.forgot.bind(this)}>
```
Comment (n-trance) [inline app/components/LoginForm/index.js]: make a simple alert for forgot until we decide what to do with it.

File: app/components/LoginForm/index.js (hunk 1 above)
Comment (dbu329) [reply app/components/LoginForm/index.js]: hmmm lets change now?

File: app/components/LoginForm/index.js (hunk 3 above)
Comment (dbu329) [reply app/components/LoginForm/index.js]: huehue i added it in, idk if it sends the email succesfully

**Pull request merged**